teacher.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'school_member'
+require_relative 'payable'
+
+class Teacher < SchoolMember
+	include Payable
+
+	def set_phase(num)
+    @phase = num
+    "Cool, I've always wanted to teach phase #{num}!"
+  end
```

Comment:
Indentation may seem trivial, as the program works whether you're following standard conventions or not.  But when you're working on a large project with lots of other people, you want to know that everyone is following the same style guide.  Even just looking back on your own code, you'll prefer well formatted code over poorly formatted code!  And any workplace with code reviews in place will require that you follow their style guide, so get used to it now.